poetry.lock

```diff
@@ -1040,8 +1018,8 @@ python-versions = "*"
 
 [[package]]
 name = "mistune"
-version = "0.8.4"
-description = "The fastest markdown parser in pure Python"
+version = "2.0.4"
```

Comment:
Are all the other changes in this file a consequence of updating mistune?

Reply:
It looks like it - I believe that it's generally recommended to run `poetry update` once in a while, which will update the resolved versions of your dependencies to the latest that matches the semver range, but these are all from just running `poetry add mistune`